src/client/script-tag-config.coffee

```diff
@@ -56,9 +56,9 @@ initParams = {
   embed: { default: true, transform: transform.bool },
   autoStart: { default: true, transform: transform.bool },
   accessToken: { default: null }
+  customerId: { default: null }
```

Comment:
Useful to go through them to have default values. We could also be more lenient and include what's not whitelisted.